src/databased/runtime.rs

```diff
@@ -97,11 +97,9 @@ impl Runtime {
                     expected_counterparty_node_id: state.connected_counterparty_node_id.clone(),
                 };
                 debug!("{} | setting checkpoint info entry", swap_id.swap_id());
-                let mut info_encoded = vec![];
-                let _info_size = info.strict_encode(&mut info_encoded);
-                self.database.set_checkpoint_info(&swap_id, &info_encoded)?;
+                self.database.set_checkpoint_info(&swap_id, &info)?;
 
-                debug!("setting swap checkpoint");
+                debug!("{} | setting swap checkpoint", swap_id.swap_id());
```

Comment:
Style: IMO no need for colors in level below `info`

Reply:
I appreciate the colors when running the swap tests locally with debug logs.